pinot-common/src/main/java/org/apache/pinot/common/function/scalar/DateTimeFunctions.java

```diff
@@ -306,7 +306,7 @@ public static long now() {
    *           "-P6H3M"    -- parses as "-6 hours and -3 minutes"
    *           "-P-6H+3M"  -- parses as "+6 hours and -3 minutes"
    */
-  @ScalarFunction
+  @ScalarFunction(nullableParameters = true)
```

Comment:
`ago(periodString)` does not handle null because it cannot return null (since return type is primitive `long`, not wrapper `Long`). Anyway, removed the annotation.